src/components/views/Registration.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { api, handleError } from "helpers/api";
+import User from "models/User";
+import { useNavigate } from "react-router-dom";
+import { Button } from "components/ui/Button";
+import "styles/views/Register.scss";
+import BaseContainer from "components/ui/BaseContainer";
+import PropTypes from "prop-types";
+
+/*
+It is possible to add multiple components inside a single file,
+however be sure not to clutter your files with an endless amount!
+As a rule of thumb, use one file per component and only add small,
+specific components that belong to the main one in the same file.
+ */
+const FormField = (props) => {
+  return (
+    <div className="register field">
+      <label className="register label">{props.label}</label>
+      {props.type === "password" ? (
+        <input
+          className="register input"
+          placeholder="enter here.."
+          type="password"
+          value={props.value}
+          onChange={(e) => props.onChange(e.target.value)}
+        />
+      ) : (
+        <input
+          className="register input"
+          placeholder="enter here.."
+          value={props.value}
+          onChange={(e) => props.onChange(e.target.value)}
+        />
```

Comment:
code smell: duplicate code
-> create something like a login field componenent for that, because is always the same for register/login just add param for psw

Reply:
Not relevant for now: can be done later if time